packages/main/src/plugin/image-registry.ts

```diff
@@ -643,7 +644,11 @@ export class ImageRegistry {
       // find the manifest corresponding to our platform
       const matchedManifest = this.getBestManifest(
         // eslint-disable-next-line @typescript-eslint/no-explicit-any
-        parsedManifest.manifests.filter((m: any) => m.mediaType === 'application/vnd.oci.image.manifest.v1+json'),
+        parsedManifest.manifests.filter(
+          (m: any) =>
```

Comment:
unknown ?

Reply:
it'll need some refactoring also it can't use unknown as next line is using `m.mediaType`